### PR TITLE
Add All Products endpoint

### PR DESCRIPTION
**Pull request by dvnishshanka**
I added the all products endpoint to the BE.
As the products json is not yet completed dummy file was added and used.
Nodemon was installed as it automatically rerun when there is a change in the file.

Front end integration was also done.

### CODE REVIEW

**Comment by 006080** [comment]
it is the product.json file in the folder "server"

**Comment by 006080** [comment]
should be approved

**Comment by dvnishshanka** [comment]
Hi, Since we have several things to clarify in the products.json file I feel we can review this and replace the products.json file with proper data once it is ready. What do you think @006080 @BajalanIman

**Comment by matus-vacula** [comment]
Let's rename products to places

File: src/App.jsx [hunk 1]
```diff
@@ -4,47 +4,24 @@ import ProductCard from "./components/ProductCard/ProductCard";
 import PersonProfile from "./components/PersonProfile/PersonProfile";
```
Comment (Konzisam) [inline src/App.jsx:4]: There is an import  "Person profile" which doesnt exist anymore in main

Comment (Konzisam) [reply src/App.jsx:4]: and the calendar toggle import is missing

Comment (dvnishshanka) [reply src/App.jsx]: Yes. You are correct. I also got the same errors. Now the main is fixed. I updated the branch to the recent main. So now it should work.

**Review by Konzisam — APPROVED** [review]

**Comment by 006080** [comment]
I merged it as I needed it now ;)

**Pull request merged**